Makefile

```diff
@@ -48,7 +48,7 @@ bin-all: bin-conda bin-docker-machine bin-kompose bin-kubectl \
 
 #<<<
 bin-conda: mkdir
-	echo "    started bin-conda" >> ~/progress.txt
+	echo "    started bin-conda" >> $(WORKING_DIR)/progress.txt
```

Comment:
There is a global $DIR variable when calling through Makefile which refers to the project directory

Reply:
Also maybe we can consider moving the progress.txt logs under $DIR/tmp/logs/session-<timestamp>.txt this way we can easily see multiple runs as logfiles

Reply:
Also please take a look at the TODO in line 3, it would be nice to toggle this kind of debugging with a DEBUG=true flag in .env

Reply:
I'm not sure that, at least the way I'm using progress.txt in my setup scripts, that this is solely a debugging log. Scripts sometimes rely on "finished xxxx" being present in order to proceed. Still, I will move it to $DIR/tmp/logs, as that is a better location for it anyway.